src/app/api/auth/google/callback/route.ts

```diff
@@ -90,6 +90,30 @@ export async function GET(request: Request): Promise<Response> {
       };
       await db.insert(accounts).values(account);
     }
+    // user exists but does not have google provider
+    else if (!new Set(user.providers as Array<Provider>).has(Provider.google)) {
+      console.log("b");
```

Comment:
Check these console.logs

Reply:
Fixed, they were results of duplicate code after fixing merge conflicts.